src/manifolds/Rotations.jl

```diff
@@ -230,25 +232,33 @@ function exp!(M::Rotations{4}, y, x, v)
         a₂ = (sincα - (1 - r) / 2 * cosα) * c
         a₃ = (e + (1 - r) * (e - sincα / 2)) * c
     end
-
-    v² = v * v
-    y .= a₀ .* x .+ x * (a₁ .* v .+ a₂ .* v² .+ a₃ .* (v² * v))
-    return y
+    v² = X * X
+    q .= a₀ .* p .+ p * (a₁ .* X .+ a₂ .* v² .+ a₃ .* (v² * X))
+    return q
 end
 
-flat!(M::Rotations, v::CoTFVector, x, w::TFVector) = copyto!(v, w)
+flat!(M::Rotations, ξ::CoTFVector, p, X::TFVector) = copyto!(ξ, X)
 
-function get_coordinates(M::Rotations, x, v, B::ArbitraryOrthonormalBasis) where {N}
-    T = Base.promote_eltype(x, v)
-    return vee(M, x, v) .* sqrt(T(2))
+function get_coordinates(M::Rotations, p, X, B::ArbitraryOrthonormalBasis) where {N}
+    T = Base.promote_eltype(p, X)
+    return vee(M, p, X) .* sqrt(T(2))
 end
 
-function get_vector(M::Rotations, x, v, B::ArbitraryOrthonormalBasis) where {N}
-    T = Base.promote_eltype(x, v)
-    return hat(M, x, v) ./ sqrt(T(2))
+function get_vector(M::Rotations, p, X, B::ArbitraryOrthonormalBasis) where {N}
+    T = Base.promote_eltype(p, X)
+    return hat(M, p, X) ./ sqrt(T(2))
 end
 
-function hat!(M::Rotations{2}, Ω, x, θ::Real)
+@doc raw"""
+    hat(M::Rotations, p, ω)
+
+Convert the unique tangent vector components $\omega$ at point `p` on [`Rotations`](@ref)
```

Comment:
While the notation `ω` and `Ω` for `SO(n)` is common due to its use for angular velocity in physics, we should probably here adopt the same notation we use elsewhere, that is `Xⁱ` and `X`.